baseroms/gc-eu-mq-dbg/checksum-compressed.md5

```diff
@@ -1 +1 @@
-5831385a7f216370cdbea55616b12fed  build/gc-eu-mq-dbg/oot-gc-eu-mq-dbg-compressed.z64
+9704bc98c20c20319cc1633fe02b6fc7  build/gc-eu-mq-dbg/oot-gc-eu-mq-dbg-compressed.z64
```

Comment:
To be clear, this changed because of fixing the compressed ranges wrt the buffers segment, right?

Reply:
Yeah, and I believe in the new version the compressed rom segments are in a different order too (they match the sorted dmadata order rather than the unsorted order)